### PR TITLE
Feature/Boost Logger Implementation

### PR DESCRIPTION
**Pull request by ShobhitAd**

### CODE REVIEW

File: modules/configuration/base_config.lua [hunk 1]
```diff
@@ -46,6 +46,9 @@ config.webengineUniqueId = "5lNGKj5n63Hf8QjZPzS7X5yjZWYD0nu6IhKtyQOFRuUZkbY8KOdF
 
 --- Logs and Reports
 --
+
+-- Flag which defines the logger name (BOOST, LOG4CXX)
+config.loggerName = "BOOST"
```
Comment (iCollin) [inline modules/configuration/base_config.lua:51]: I think we should leave the default as `LOG4CXX`

**Review by iCollin — APPROVED** [review]

File: tools/runners/parallels.sh [hunk 2]
```diff
@@ -190,6 +190,7 @@ function mktempdir {
 
   cp $_sdl_prepared/bin/hmi_capabilities.json $tmpdirname/bin/
   cp $_sdl_prepared/bin/log4cxx.properties $tmpdirname/bin/
+  cp $_sdl_prepared/bin/boostlogconfig.ini $tmpdirname/bin/
```
Comment (iCollin) [inline tools/runners/parallels.sh:193]:
```suggestion
  cp $_sdl_prepared/bin/boostlogconfig.ini $tmpdirname/bin/ 2>/dev/null
```

Using ATF against a Core build without boostlog prints this error a lot, can we suppress?

```
cp: cannot stat '/ram/prepared_env/sdl/bin/boostlogconfig.ini': No such file or directory
```

**Review by iCollin — CHANGES_REQUESTED** [review]

File: modules/configuration/base_config.lua (hunk 1 above)
Comment (ShobhitAd) [reply modules/configuration/base_config.lua]: Agreed, changed in https://github.com/smartdevicelink/sdl_atf/pull/216/commits/314ae47670017992a0038790320432e3649187d5

File: tools/runners/parallels.sh (hunk 2 above)
Comment (ShobhitAd) [reply tools/runners/parallels.sh]: Added in https://github.com/smartdevicelink/sdl_atf/pull/216/commits/314ae47670017992a0038790320432e3649187d5

**Review by iCollin — APPROVED** [review]

**Pull request merged**